thundra/integrations/botocore.py

```diff
@@ -242,41 +218,37 @@ def __init__(self):
 
     def get_operation_name(self, wrapped, instance, args, kwargs):
         _, request_data = args
-        return request_data['StreamName'] if 'StreamName' in request_data else Constants.AWS_SERVICE_REQUEST
-        # return 'kinesis'
+        return request_data['StreamName'] if 'StreamName' in request_data else constants.AWS_SERVICE_REQUEST
 
     def getRequestType(self, string):
-        if string in Constants.KinesisRequestTypes:
-            return Constants.KinesisRequestTypes[string]
-        return Constants.AWS_SERVICE_REQUEST
-
-    def inject_span_info(self, scope, wrapped, instance, args, kwargs, response,
-                         exception):
+        if string in constants.KinesisRequestTypes:
+            return constants.KinesisRequestTypes[string]
+        return constants.AWS_SERVICE_REQUEST
```

Comment:
Return empty string instead of `AWS_SERVICE_REQUEST` constant if the stream name is not available

Reply:
But be sure that in case of empty stream name, operation name is not empty but `AWS_SERVICE_REQUEST` constant